tool/grind.dart

```diff
@@ -5,8 +5,28 @@ import 'dart:io';
 
 import 'package:grinder/grinder.dart';
 
-main(args) => grind(args);
+import 'package:unscripted/unscripted.dart';
 
+main(args) {
+
+  addTask(new GrinderTask(
```

Comment:
I basically just added this task as an example of how to use the imperative API.  Though this task could be useful if we also make it update the version constant in the code simultaneously.  This is also the use case @sethladd was wanting this feature for.